vault/resource_gcp_auth_backend_test.go

```diff
@@ -113,3 +174,41 @@ resource "vault_gcp_auth_backend" "test" {
 }
 `, credentials, path)
 }
+
+func testGCPAuthBackendConfig_update(path, credentials string) string {
+	return fmt.Sprintf(`
+variable "json_credentials" {
+  type = string
+  default = %q
+}
+
+resource "vault_gcp_auth_backend" "test" {
+  path                          = %q
+  credentials                   = var.json_credentials
+  custom_endpoint {
+    api     = "www.googleapis.com"
+    iam     = "iam.googleapis.com"
+    crm     = "cloudresourcemanager.googleapis.com"
+    compute = "compute.googleapis.com"
+  }
+}
+`, credentials, path)
+}
+
+func testGCPAuthBackendConfig_update_partial(path, credentials string) string {
+	return fmt.Sprintf(`
+variable "json_credentials" {
+  type = string
+  default = %q
+}
+
+resource "vault_gcp_auth_backend" "test" {
```

Comment:
nit: formatting is bit off, I'd say either stick to single space padding or format using `terraform fmt`